scripts/build_android.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+PROJECT_DIR=$(pwd)
+
+# Color definitions
+DEFAULT='\033[0m'
+RED='\033[0;31m'
+
+cd ${PROJECT_DIR}/mopro-ffi
+
+cargo build --lib \
+--target x86_64-linux-android \
+--target i686-linux-android \
+--target armv7-linux-androideabi \
+--target aarch64-linux-android
+
+for binary in target/*/*/libmopro_ffi.so; do file $binary; done
+
+mkdir -p jniLibs/arm64-v8a/ && \
```

Comment:
I think it depends on the device/simulator 
like the simulator is arm64-v8a, then we need to add the target `aarch64-linux-android`

<img width="824" alt="截圖 2023-12-04 下午6 26 26" src="https://github.com/oskarth/mopro/assets/17507085/b79c0119-9d6e-4a34-be47-4c37e90ab2f5">

what do you mean by using shared library?
Actually we don't have a real android device so I haven't tried

Reply:
.so is a shared library, it might be problematic if it expects certain cryptographic libraries to be available on Android when they might not be.

> real device

ok, we can merge as progress and then create issue for supporting and testing on real device, wdyt?